agent/config/parse.go

```diff
@@ -117,6 +117,22 @@ func parseNumImagesToDeletePerCycle() int {
 	return numImagesToDeletePerCycle
 }
 
+func parseImagePullBehavior() ImagePullBehaviorType {
+	ImagePullBehaviorString := os.Getenv("ECS_IMAGE_PULL_BEHAVIOR")
```

Comment:
should we .ToLowerCase() the string?

Reply:
I think we use `trimWhitespace ` to format every env var string and we should not do extra work outside of this method:
https://github.com/aws/amazon-ecs-agent/blob/159ae5c3067e0a0d7387ff68af2a28bc90f3baaa/agent/config/config.go#L187